### PR TITLE
Fix: invalid login feedback

### PR DESCRIPTION
**Pull request by Orick08**
## What?
Shows a message to the user if the email or password is wrong. This solves the issue #181.
## Why?
In the current version, if the credentials are wrong it only shows the login again without any feedback.
## How?
I add a boolean flag on the page, but for further forms validations (issue #72) maybe is better to discuss a library to do this automatically like formik-yup.
## Testing?
I tested with invalid credentials, the only misbehave is that if the server is down and cant communicate it shows the same message.
## Screenshots
Screen Shot 2021-12-17 at 12 08 51 PM

### CODE REVIEW

**Review by olivermrbl — APPROVED** [review]
LGTM - thanks for your contribution! ❤️

File: src/pages/login.js [hunk 1]
```diff
@@ -29,6 +31,9 @@ const IndexPage = () => {
       .finally(() => {
         setLoading(false)
       })
+      .catch(e => {
+        setIsInvalidLogin(true)
+      })
```
Comment (olivermrbl) [inline src/pages/login.js:36]: Tiny thing. I think we should change to have the `finally`-block as the last part of the promise chain.

**Review by olivermrbl — CHANGES_REQUESTED** [review]

File: src/pages/login.js (hunk 1 above)
Comment (Orick08) [reply src/pages/login.js]: Yep, no problem, done

**Comment by olivermrbl** [comment]
Thanks for the fix @Orick08 ! We included a very similar fix in #325 so closing this one :)

**Pull request closed without merging**